src/services/billing/cost-management/cost-dashboard/modules/CostDashboardPeriodSelectDropdown.vue

```diff
@@ -112,7 +130,23 @@ export default {
             state.customRangeModalVisible = false;
         };
 
-        watch(() => props.fixedPeriod, () => {
+        const handleSelectedFixDate = async (isSelected: boolean) => {
+            try {
+                const periodType = isSelected ? 'FIXED' : 'AUTO';
+                const fixedPeriod = isSelected ? state.period : {};
+                await SpaceConnector.client.costAnalysis.dashboard.update({
+                    dashboard_id: props.dashboardId,
+                    period_type: periodType,
+                    period: fixedPeriod,
+                });
+                emit('update:fixed-period', periodType, fixedPeriod);
```

Comment:
```suggestion
                emit('update:fixed-period', periodType);
                 emit('update:period', fixedPeriod);
```

뷰에서 'update:xxx' 이벤트는 prop을 양방향 바인딩할 수 있도록하는 sync 기능을 위해 사용되는 이벤트명이므로, 관련된 prop만 이벤트 파라미터로 넘기는게 좋을 것 같습니다.